src/igvfd/tests/fixtures/schemas/treatment.py

```diff
@@ -53,13 +59,29 @@ def treatment_v2(treatment_chemical):
 
 
 @pytest.fixture
-def treatment_ntr(testapp):
+def treatment_ntr(testapp, lab, award):
     item = {
         'treatment_term_id': 'NTR:100',
         'treatment_term_name': 'interferon gamma',
         'treatment_type': 'chemical',
         'amount': 10,
         'amount_units': 'mM',
-        'purpose': 'perturbation'
+        'purpose': 'perturbation',
+        'award': award['@id'],
+        'lab': lab['@id'],
+        'depletion': False
     }
     return testapp.post_json('/treatment', item, status=201).json['@graph'][0]
+
+
+@pytest.fixture
+def treatment_v3(treatment_chemical):
+    item = treatment_chemical.copy()
```

Comment:
The fixture should 'pop' depletion property as well to test the upgrade

Reply:
popped.